packages/sitecore-jss-nextjs/package.json

```diff
@@ -42,12 +41,14 @@
     "enzyme-adapter-react-16": "^1.14.0",
     "jsdom": "^15.1.1",
     "mocha": "^8.1.3",
+    "next": "^10.0.2",
```

Comment:
Would you mind updating next version in the sample app as well? Also, 10.0.3 just dropped overnight :)

Reply:
Changed